src/js/pages/organizer/Cards.jsx

```diff
@@ -1,9 +1,249 @@
 import React from "react";
+import {BootstrapTable, TableHeaderColumn} from 'react-bootstrap-table';
+import BrandStore from "stores/BrandStore";
 
 export default class Cards extends React.Component {
+ 
+  constructor() {
+    super();
+    this.state = {
+      cards: [],
+      locations: [],
+      contacts: [],
+      sponsors: [],
+    };
+
+    BrandStore.getAll();
+  }
+
+  componentWillMount() {
+    BrandStore.on("receivedCards", this.getCards);
+    BrandStore.on("receivedContacts", this.getContacts);
+    BrandStore.on("receivedLocations", this.getLocations);
+    BrandStore.on("receivedSponsors", this.getSponsors);
+    BrandStore.on("error", this.showError);
+  }
+
+  componentWillUnmount() {
+    BrandStore.removeListener("receivedCards", this.getCards);
+    BrandStore.removeListener("receivedContacts", this.getContacts);
+    BrandStore.removeListener("receivedLocations", this.getLocations);
+    BrandStore.removeListener("receivedSponsors", this.getSponsors);
+    BrandStore.removeListener("error", this.showError);
+  }
+
+  showError = () => {
+    console.log(BrandStore.error)
+  }
+ 
+  generateCard = (row) => {
+    var updatedCard = {
+      id: -1,
+      data: "",
+    }
+    for (const prop in row) {
```

Comment:
if you cared you could look at the extend function to simplify this https://api.jquery.com/jquery.extend/

You dont have to do it, just helpful stuff